nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -927,6 +976,7 @@ private Description checkForReadBeforeInit(ExpressionTree tree, VisitorState sta
     if (!symbol.getKind().equals(ElementKind.FIELD)) {
       return Description.NO_MATCH;
     }
+
```

Comment:
extra blank line?

Reply:
Intentional, to separate two logical checks: a) is this a field read? b) if it's a static field, is the enclosing initializer static?

But I don't feel too strongly about it either way, could remove the new line.

Reply:
We can leave it